_meta/ecs-migration.yml

```diff
@@ -35,42 +38,42 @@
 - from: context.system.platform
   to: host.os.platform
 
-#- from: context.request.method
-#  to: http.request.method
-#
-#- from: context.request.http_version
-#  to: http.version
-#
-#- from: context.request.body
-#  to: http.request.body
-#  index: false
-#
-## Not in ECS
-#- from: context.request.cookies
-#  to: http.request.headers.cookies.parsed
-#  index: false
-#
-## Not in ECS
-#- from: context.request.headers.cookie
-#  to: http.request.headers.cookies.original
-#  index: false
-#
-#- from: context.request.headers.user-agent
-#  to: http.request.headers.user-agent
-#  index: false
-#
-#- from: context.request.headers.content-type
-#  to: http.request.headers.content-type
-#  index: false
-#
-#- from: context.request.env
-#  to: http.request.env
-#  index: false
-#
-#- from: context.request.socket
-#  to: http.request.socket
-#  index: false
-#
+- from: context.request.method
+  to: http.request.method
+
+- from: context.request.http_version
+  to: http.version
+
+- from: context.request.body
+  to: http.request.body.content
```

Comment:
As we don't restrict this to keywords it should be `http.request.body.content.text`

Reply:
actually, in the intake `body` is either a string or an object (naturally), so this is the same case as `request.env` 
i am leaning towards leaving it as is

Reply:
fine with me, thanks for looking into it